app/frontend/components/domains/requirements-library/requirement-block-accordion.tsx

```diff
@@ -110,11 +125,19 @@ export const RequirementBlockAccordion = observer(function RequirementBlockAccor
                 />
               )}
               {isEditable && renderEdit?.()}
-              <AccordionIcon color={"text.primary"} />
+              <button>
+                <AccordionIcon color={"text.primary"} />
+              </button>
```

Comment:
this is fine but for next time chakra has an `IconButton` component that can be used if we want to keep it consistent and use chakra for everything possible.